integration-php7.2/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM nextcloudci/php7.2:php7.2-13
```

Comment:
Why do you used a fixed version? I was wondering about that when adding xdebug again to some images. It's a nightmare ;) Given that images are only built on tagging using latest should be fine.

Reply:
we didn't use latest tag. I added it in some builds last month, but lots are missing

Reply:
Ah. I thought the latest tag is automatically done by docker ;) The good thing is you already added it to https://hub.docker.com/r/nextcloudci/php7.2/tags so we can use it :tada: